adapters/oidc/js/src/main/resources/login-status-iframe.html

```diff
@@ -81,7 +81,16 @@
 
     function getCookie()
     {
-        var name = 'KEYCLOAK_SESSION=';
+        var cookie = getCookieByName('KEYCLOAK_SESSION');
+        if (cookie === null) {
+            cookie = getCookieByName('KEYCLOAK_SESSION_LEGACY');
```

Comment:
There is no need to re-assign `cookie` here, instead the result of `getCookieByName('KEYCLOAK_SESSION_LEGACY')` can be returned immediately.

Reply:
@jonkoops The legacy cookie is basically the same cookie as we set before this PR, i.e. it doesn't have the SameSite attribute. Chrome >80 won't send cookies in iframes that have no SameSite attribute set (which defaults to SameSite=Lax). In other words Chrome won't see the legacy cookie and at the same time some other browsers – especially older Apple browsers – reject the non-legacy cookie, so they see only the legacy cookie.

Reply:
@jonkoops I'm sorry, I got it wrong the first time. You're correct, technically there's no need to re-assign the variable. But to me it seems more clear to have only one `return` but I don't really insist on it. Do you think this impacts performance?

Reply:
No apology needed, hahah. It's a code-style thing, so I would not worry about it too much.

In the JavaScript community early returns are generally preferred over re-assigning a variable. This is emphasised by the preferred usage of `const` over `var` and  `let` in modern codebases.

Reply:
Ok, fair enough. ;) Updated per your suggestion.